ingest/src/lib.rs

```diff
@@ -169,3 +169,63 @@ pub async fn start_stream(provider_config: ProviderConfig, mut query: Query) ->
 
     Ok(Box::pin(stream))
 }
+
+
+#[cfg(test)]
+mod tests {
+
+    use crate::svm::*;
+    use parquet::arrow::ArrowWriter;
+    use std::fs::File;
+    use super::*;
+
+    #[tokio::test]
+    #[ignore]
+    async fn simple_svm_start_stream() {
```

Comment:
I'm creating these tests to help me with the editing process. They are not meant to cover all possibilities, but they give me room to edit them and test different parts of the code.